project/GenerateOneToManies.scala

```diff
@@ -84,6 +85,9 @@ class OneToManies${n}SQL[A, $bs, E <: WithExtractor, Z](
     extends SQL[Z, E](statement, rawParameters)(SQL.noExtractor[Z]("one-to-many extractor(one(RS => A).toManies(RS => Option[B1]...)) is specified, use #map((A,B) =>Z) instead."))
     with AllOutputDecisionsUnsupported[Z, E] {
 
+  def copy(statement: String = statement, rawParameters: Seq[Any] = rawParameters, one: WrappedResultSet => A = one, $resultSetToOptionsWithDefaultValue, extractor: (A, $seq) => Z = extractor): OneToManies${n}SQL[A, $bs, E, Z] = {
```

Comment:
The query is not completely different, but partially modified (ex. append a condition to the `where` clause). So it seems not quite unreasonable for me, how do you think?